slider/src/test/java/uk/gov/gchq/gaffer/slider/GafferSliderDeployer.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.slider;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.core.client.security.tokens.PasswordToken;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.accumulo.core.security.SystemPermission;
+import org.apache.slider.api.ClusterDescription;
+import org.apache.slider.client.SliderClient;
+import org.apache.slider.core.conf.ConfTree;
+import org.apache.slider.core.persist.ConfTreeSerDeser;
+import org.apache.slider.funtest.framework.AgentCommandTestBase;
+import org.apache.slider.funtest.framework.CommandTestBase;
+import org.apache.slider.funtest.framework.SliderShell;
+import org.apache.slider.test.SliderTestUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.gchq.gaffer.accumulostore.AccumuloProperties;
+import uk.gov.gchq.gaffer.accumulostore.SingleUseAccumuloStore;
+import uk.gov.gchq.gaffer.commonutil.StreamUtil;
+import uk.gov.gchq.gaffer.slider.util.*;
+import uk.gov.gchq.gaffer.store.StoreProperties;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.net.URISyntaxException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Deploys a gaffer-slider instance using the Slider functional testing library
+ */
+public class GafferSliderDeployer extends AgentCommandTestBase {
```

Comment:
Sorry I've just released that is just because it tries to run the integration tests. Just seen your note in the readme about running the integration tests. Perhaps we can just skip the integration tests from the standard "mvn clean install".

Doing "mvn clean package" works as this doesn't run the integration tests.